### DISCUSSION TITLE
could disable GetColumnName(xx) parameter performColumnAliasNormalization?

### DISCUSSION THREAD

**Discussion by sun51586** [MoonStorm/FastCrud#173] [Q&A]
thanks for your work.
Entity property with ColumnAttribute,for example:
public class Test{
  [Column("USER_NAME")]
  public virtual string UserName{get;set;}
}

Use Dapper.FastCrud connection.Find<Test4Dev>() is ok,
but when use Dapper connection.Query<Test>("select * from test") the UserName will return null.

I can new CustomPropertyTypeMap and use SqlMapper.SetTypeMap() to handle connection.Query<Test>(sql),
but Dapper.FastCrud is return null because of StatementSqlBuilder.cs->GetColumnName()->performColumnAliasNormalization is true.

so could you fix this with add a switch in the GetColumnName() method? like follow codes,thanks very much. 

File:\Dapper.FastCRUD-master\Dapper.FastCrud\SqlBuilders\StatementSqlBuilder.cs
public string GetColumnName(PropertyMapping propMapping, string tableAlias, bool performColumnAliasNormalization)
{
//could add this code???
if(!OrmConfiguration.EnableColumnAlias){
     performColumnAliasNormalization=false;
}
//Original code
}

EnableColumnAlias can define in OrmConfiguration class.

**Comment by MoonStorm** [comment]
Until we'll come up with a better idea (most likely a format specifier), I mostly use this in my custom query building code:
```c#
  OrmConfiguration.GetSqlBuilder<User>().ConstructColumnEnumerationForSelect()
```
